src/views/NotificationButton.tsx

```diff
@@ -124,12 +124,28 @@ class NotificationButton extends ComponentEx<IProps, IComponentState> {
       });
     }
 
+    const isRunningBgProcess = 
+      notifications
+      .filter((x) => x.type === 'activity' && x.progress === undefined)
+      .length >= 1 && combinedProgress.length == 0;
+    
+    if (isRunningBgProcess) {
```

Comment:
This is quite hacky, I really don't like it.
The look of the radial progress indicator is implemented by RadialProgress and _its_ css.
That's a fairly decent, reusable component.

The style of the "activity" indication you now have as part of the notification button and its css, making it not reusable and the styling is spread out over multiple files (the filled out ring being 1/4 of the circle is here, the animation speed in the css)
Please clean this up, make this spinner part a feature of the RadialProgress component